// THIS FILE IS AUTOMATICALLY GENERATED. DO NOT EDIT.

package datapipeline

const (

	// An internal service error occurred.
	ErrCodeInternalServiceError = "InternalServiceError"

	// The request was not valid. Verify that your request was properly formatted,
	// that the signature was generated with the correct credentials, and that you
	// haven't exceeded any of the service limits for your account.
	ErrCodeInvalidRequestException = "InvalidRequestException"

	// The specified pipeline has been deleted.
	ErrCodePipelineDeletedException = "PipelineDeletedException"

	// The specified pipeline was not found. Verify that you used the correct user
	// and account identifiers.
	ErrCodePipelineNotFoundException = "PipelineNotFoundException"

	// The specified task was not found.
	ErrCodeTaskNotFoundException = "TaskNotFoundException"
)
